pyserini/eval/trec_eval.py

```diff
@@ -52,90 +52,100 @@
 # the JVM has already loaded.
 from pyserini.search import get_qrels_file
 
-cmd_prefix = ['java', '-cp', jar_path, 'trec_eval']
-
-args = sys.argv
-
-# Option to discard non-judged hits in run file
-judged_docs_only = ''
-judged_result = []
-cutoffs = []
-
-if '-remove-unjudged' in args:
-    judged_docs_only = args.pop(args.index('-remove-unjudged'))
-
-if any([i.startswith('judged.') for i in args]):
-    # Find what position the arg is in.
-    idx = [i.startswith('judged.') for i in args].index(True)
-    cutoffs = args.pop(idx)
-    cutoffs = list(map(int, cutoffs[7:].split(',')))
-    # Get rid of the '-m' before the 'judged.xxx' option
-    args.pop(idx-1)
-
-temp_file = ''
-
-if len(args) > 1:
-    if not os.path.exists(args[-2]):
-        args[-2] = get_qrels_file(args[-2])
-    if os.path.exists(args[-1]):
-        # Convert run to trec if it's on msmarco
-        with open(args[-1]) as f:
-            first_line = f.readline()
-        if 'Q0' not in first_line:
-            temp_file = tempfile.NamedTemporaryFile(delete=False).name
-            print('msmarco run detected. Converting to trec...')
-            run = pd.read_csv(args[-1], sep='\s+', header=None, names=['query_id', 'doc_id', 'rank'])
-            run['score'] = 1 / run['rank']
-            run.insert(1, 'Q0', 'Q0')
-            run['name'] = 'TEMPRUN'
+def eval(args, query_id=None) -> float: 
```

Comment:
why just `eval`? Maybe add some clarity of what function it is actually by saying trec_eval or something